WorksheetB

```diff
@@ -0,0 +1,31 @@
+Word Game Psuedocode
+
+Likeness : int
+PlayerInput : string
+ActualWord = House
+Guesses : int
+
+Print a list of words that includes the chosen word house and other similar words
+Print " guess my word"
+Split ActualWord into 5 separate strings with 1 string for each letter
+
+While Guesses <= 4
+Read PlayerInput
```

Comment:
The code inside the while loop should be indented for readability -- especially since there is no "end while" to indicate where the loop ends.